setup/create_binaries.sh

```diff
@@ -3,8 +3,8 @@ set -e
 venv_name=venv
 rm -rf $venv_name
 virtualenv $venv_name
-$venv_name/bin/pip install "git+https://github.com/pyinstaller/pyinstaller.git@12e40471c77f588ea5be352f7219c873ddaae056#egg=pyinstaller"
+$venv_name/bin/pip install pyinstaller==3.1.1
 $venv_name/bin/pip install .
-$venv_name/bin/pyinstaller -F --runtime-hook=setup/binary-hook.py setup/bin/dusty
+$venv_name/bin/pyinstaller -F --runtime-hook=setup/binary-hook.py setup/dusty.spec
```

Comment:
previously, pyinstaller generated a specs file on the fly from inspecting our source. this created a specs file very similar to the one I've committed. the only difference is that now I need to include my resource files, which pyinstaller can't do automatically (afaict).

i copied the generated specs file, added my resources files to the analysis object, then told pyinstaller to use my specs instead of trying to generate them